Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -278,29 +206,8 @@ inline void AssignVSOutputMembers(T& object, const char* a, const char* b, u32 t
 // As a workaround, we interpolate at the centroid of the coveraged pixel, which
 // is always inside the primitive.
 // Without MSAA, this flag is defined to have no effect.
-inline const char* GetInterpolationQualifier(bool msaa, bool ssaa,
-                                             bool in_glsl_interface_block = false, bool in = false)
-{
-  if (!msaa)
-    return "";
-
-  // Without GL_ARB_shading_language_420pack support, the interpolation qualifier must be
-  // "centroid in" and not "centroid", even within an interface block.
-  if (in_glsl_interface_block && !g_ActiveConfig.backend_info.bSupportsBindingLayout)
-  {
-    if (!ssaa)
-      return in ? "centroid in" : "centroid out";
-    else
-      return in ? "sample in" : "sample out";
-  }
-  else
-  {
-    if (!ssaa)
-      return "centroid";
-    else
-      return "sample";
-  }
-}
+const char* GetInterpolationQualifier(bool msaa, bool ssaa, bool in_glsl_interface_block = false,
```

Comment:
```suggestion
std::string_view GetInterpolationQualifier(bool msaa, bool ssaa, bool in_glsl_interface_block = false,
```

Reply:
It's probably nearly irrelevant but I noticed that result of this function is passed to a `string_view` argument, so using a string view here may tidy things up a bit.

Reply:
I'm waiting until all relevant usage cases with the ShaderCode generators are converted over to WriteFmt, so that we're using an API that safely handles custom types like string_view before making this change.